bucket/kubeadm.json

```diff
@@ -0,0 +1,27 @@
+{
+    "homepage": "https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm/",
+    "description": "Kubernetes node provisioning tool.",
+    "license": "Apache-2.0",
```

Comment:
Where did you get this information?

Reply:
https://github.com/kubernetes/kubernetes/blob/master/LICENSE
kubeadm is an application that is part of the kubernetes/kubernetes repo: https://github.com/kubernetes/kubernetes/tree/master/cmd/kubeadm